packages/ckeditor5-dev-utils/lib/compiler/resolveimportpath.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const path = require( 'path' );
+const fs = require( 'fs' );
+const getWorkspaceRelativePathInfo = require( './getworkspacerelativepathinfo' );
+
+/**
+ * Looks for the imported path in context of the requester path and returns informations about it when the request matches some file.
+ *
+ * @param {String} requesterPath Path to the requester.
+ * @param {String} importPath Path to the imported file.
+ * @param {Array.<String>} packagePaths Array of node_modules directories and packages where packages will be looking for.
```

Comment:
An array of directories where the algorithm will look for packages. You can either pass path to directories containing packages or to a package itself. If a name of the package to import matches the name of the passed directory, then this path will be used as a path to package. If not, then packages will be looked for in the specified directory.